Server/app/Http/Requests/RegisterRequest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class RegisterRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'name'=>'required|string|max:255',
+            'email'=>'required |string|email|max:255|unique:users',
+             'password'=>[
+                'required',
+                'string',
+                // 'min:8',
```

Comment:
Nice job! but why don't we have a min limit? in order to secure our application we need to ensure that there is no easy passwords allowed for example, a, 1, 123, etc. So let's enable it.